### PR TITLE
Cloudwatch

### PR DESCRIPTION
**Pull request by arno-hartmann**
Init Cloudwatch to call lambda-function to get latest weather once a day (at 06:25)

### CODE REVIEW

File: create_sunnydays/write_cities_to_dynamodb.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    {'city': 'Windhoek', 'city_id': '1466719', 'airport' : 'WDH'}
+    ]
+
+for item in cities:
+    dict = {
+        'city' : item['city'],
+        'city_id' : item['city_id'],
+        'airport' : item['airport']
+    }
+    table_destination.put_item(Item=dict)
```
Comment (fabianschmauder) [inline create_sunnydays/write_cities_to_dynamodb.py]: maybe move it to a method

File: create_sunnydays/write_cities_to_dynamodb.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+table_destination = dynamodb.Table('destination')
+
+cities = [
+    {'city': 'London', 'city_id': '44418', 'airport' : 'LOND'}, 
+    {'city': 'Athen', 'city_id': '946738', 'airport' : 'ATH'}, 
+    {'city': 'Lissabon', 'city_id': '742676', 'airport' : 'LIS'},
+    {'city': 'Barcelona', 'city_id': '753692', 'airport' : 'BCN'},
+    {'city': 'Marseille', 'city_id': '610264', 'airport' : 'MRS'},
+    {'city': 'Windhoek', 'city_id': '1466719', 'airport' : 'WDH'}
+    ]
```
Comment (fabianschmauder) [inline create_sunnydays/write_cities_to_dynamodb.py]: i would put it into a json file and load the file when running the script.
Then you only need to change the json file when the data changes

**Review by fabianschmauder — COMMENTED** [review]
⛱️

File: create_sunnydays/write_cities_to_dynamodb.py (hunk 1 above)
Comment (arno-hartmann) [reply create_sunnydays/write_cities_to_dynamodb.py]: great idea, did it!

File: create_sunnydays/write_cities_to_dynamodb.py (hunk 2 above)
Comment (arno-hartmann) [reply create_sunnydays/write_cities_to_dynamodb.py]: done!

File: create_sunnydays/write_cities_to_dynamodb_utils.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import json
+
+dynamodb = boto3.resource('dynamodb')
+
+
+table_destination = dynamodb.Table('destination')
+
+
+def write_cities_to_dynamodb():
+    f = open('cities.json')
```
Comment (fabianschmauder) [inline create_sunnydays/write_cities_to_dynamodb_utils.py]: I would use 
`with open("test.txt", encoding = 'utf-8') as f:`

**Pull request merged**